tv_apps/browser/js/sync/settings.js

```diff
@@ -258,13 +258,16 @@
           onlogout: () => {},
           onready: () => {},
           onerror: error => {
+            this.elements.collectionHistory.checked = false;
             console.error(error);
           }
         });
       } catch(e) {}
```

Comment:
I added the catch because the mozId API is crappy and it throws when you call .watch more than once. Even if it throws, the API still fires the on* callbacks.